### PR TITLE
Feature/english strings init

### PR DESCRIPTION
**Pull request by nicksarafa**

### CODE REVIEW

**Comment by nicksarafa** [comment]
Yudis is translating our bahasya strings and we're going to push all these through later on as one complete unit...

Please hard code your strings to the vues (lol) in the meantime until we have all our strings in the appropriate files and i'll swap the references with the i18n refs

File: public/src/components/Manager/Manager.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
         </v-btn>
       </v-toolbar-items>
       <v-spacer />
-      <v-toolbar-title v-text="$route.name" />
+      <v-toolbar-title v-text="$t(`routeNames.${$route.name}`)" />
```
Comment (nicksarafa) [inline public/src/components/Manager/Manager.vue]: es6 template literal coorlating to the routeNames inside our `i18n`

File: public/src/router.js [hunk 2]
```diff
@@ -25,10 +25,11 @@ export default new Router({
     {
       path: '/manager',
       component: Manager,
+      // please set the `name` to the appropriate key inside our i18n config
```
Comment (nicksarafa) [inline public/src/router.js]: needs a better description of how the headerTitle names work

File: public/src/components/Manager/Manager.vue [hunk 3]
```diff
@@ -35,25 +35,56 @@
     </v-bottom-nav>
   </v-app>
 </template>
-
 <script>
 export default {
-  name: 'Manager',
-  created () {
+  name: "Manager",
```
Comment (roma219) [inline public/src/components/Manager/Manager.vue]: aren't we using singlequotes in js?

Comment (nicksarafa) [reply public/src/components/Manager/Manager.vue]: yeah the prettier i'm using messed this up. i'm introducing a prettierrc file in a different pull request

Comment (nicksarafa) [reply public/src/components/Manager/Manager.vue]: i merged in #62 and i'm going to re-run it against this file to get the single quotes init

Comment (nicksarafa) [reply public/src/components/Manager/Manager.vue]: fixed

**Comment by nicksarafa** [comment]
👍

**Pull request merged**